snowplow/iglu-client-embedded/schemas/com.metabase/serialization/jsonschema/1-0-1

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Serialization operation",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "database",
+    "format": "jsonschema",
+    "version": "1-0-1"
+  },
+  "type": "object",
+  "required": ["event", "source", "duration", "success"],
+  "properties": {
```

Comment:
You appear to have dropped the "event" key.

Reply:
I intended to do that! Also need to remove it from required.

That's what a guy from snowcat asked me to do back in May actually 😁